packages/react/src/components/dateInput/DateInput.stories.scss

```diff
@@ -0,0 +1,9 @@
+.odd--day,
+.odd--color {
+    background: var(--color-suomenlinna);
+}
+
+.even--day,
+.even--color {
+    background: var(--color-black-10);
+}
```

Comment:
Actually these styles aren't loaded in remote environment, at least in our demo site 😮 They do work if they are included inside a style tag in the story's return however.

Reply:
Seems like a loader issue, but maybe sorting that is out of scope and I'll move the styles inline as you suggested.